modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -76,37 +76,84 @@ public function enqueue_tracking_script() {
 		wp_enqueue_script( 'woocommerce-analytics', esc_url( $filename ), array(), null, false );
 	}
 
+	/**
+	 * Default event properties which should be included with all events.
+	 *
+	 * @return array Array of standard event props.
+	 */
+	public function get_common_properties() {
+		return array(
+			'blog_id'     => Jetpack::get_option( 'id' ),
+			'ui'          => $this->get_user_id(),
+			'url'         => home_url(),
```

Comment:
I'm curious what the url is for. We already have browser document location in `_dl` property. Are you looking for a unique way to identify sites? If so, would `blog_id` suffice?

Reply:
`home_url()` is to give a way to map to data collected by WC Tracker, which does not have `blog_id`. There have been experiments in the past with using document location, but that comes with some challenges.